plum/resolvable.py

```diff
@@ -41,6 +41,15 @@ def deliver(self, obj):
 
     def resolve(self):
         if self._obj is None:
-            raise ResolutionError("Promise was not kept.")
+            raise ResolutionError("Promise `{}` was not kept.".format(repr(self)))
```

Comment:
```suggestion
            raise ResolutionError(f"Promise `{self!r}` was not kept.")
```

Reply:
I'm ok with this, but i think the f-string syntax has only been around since python 3.6.
I thouht you were trying to support older versions?